static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain.js

```diff
@@ -4,23 +4,29 @@ define([
     'common/utils/_',
     'common/utils/config',
     'common/utils/detect',
-    'common/utils/mediator'
+    'common/utils/mediator',
+    'common/utils/template',
+    'text!common/views/commercial/outbrain.html'
 ], function (
     fastdom,
     $,
     _,
     config,
     detect,
-    mediator
+    mediator,
+    template,
+    outbrainTpl
 ) {
     var outbrainUrl = '//widgets.outbrain.com/outbrain.js',
-        $outbrain = $('.js-outbrain');
+        $outbrain   = $('.js-outbrain'),
```

Comment:
I wouldn't do this one here. This runs during page load and we have already many things to do.
I'd move them inside the `mediator` callback in `init`.